### DISCUSSION TITLE
doubt in gait reward

### DISCUSSION THREAD

**Discussion by Krishnendu1984** [rohanpsingh/LearningHumanoidWalking#6] [General]
Kindly can you let me know in reward.py file how the gait reward has been designed
how r<0.5
I am getting self._period ass 88
self._phase=random init (0,period)

**Comment by rohanpsingh** [comment]
Hi, 
I couldn't really understand your question.
I think you need to read this paper for more details:
https://arxiv.org/abs/2011.01387
